packages/opds-web-client/src/__tests__/setupEnyzme.ts

```diff
@@ -0,0 +1,4 @@
+import { configure } from 'enzyme';
+import * as Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
```

Comment:
How does this work with nothing referencing this file? Could it be combined with src/testHelper?

Reply:
Yeah, it does work. Will update.